### PR TITLE
Added ironic_inventory dependency

### PR DESCRIPTION
**Pull request by chalupaul**
Adds the tool for adding ironic hosts via excel file.

### CODE REVIEW

**Comment by BjoernT** [comment]
I was under the impression we add a Python scripts and playbook ?

**Comment by chalupaul** [comment]
Fixed the typo. Good catch.

File: playbooks/install-ironic-helpers.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
```
Comment (BjoernT) [inline playbooks/install-ironic-helpers.yml]: This play does the same as the other ?

File: playbooks/vars/main.yml [hunk 2]
```diff
@@ -20,6 +20,8 @@ ops_pip_dependencies:
   - shade
   - python-openstackclient
   - python-neutronclient
+  - ironic_importer
```
Comment (BjoernT) [inline playbooks/vars/main.yml]: I think it should be ironic-importer, at least that's what it's called on pypi.
Also the version is set to python 3.5 so pip under python 2, which is still the OSA default version will fail now

**Comment by chalupaul** [comment]
not gonna work on python2.

**Comment by BjoernT** [comment]
FYI I'll fix this in the next 1-2 weeks

**Pull request closed without merging**